OsmAnd/src/net/osmand/plus/dashboard/DashboardOnMap.java

```diff
@@ -1032,6 +1042,12 @@ void onDetach(DashBaseFragment dashBaseFragment) {
 		}
 	}
 
+	public void onMapSettingsUpdated() {
+		if (configureMapMenu != null) {
+			configureMapMenu.updateMenuItem(MAP_STYLE_ID);
```

Comment:
Here we should not use MAP_STYLE_ID. We don't know which item will be updated and we don't know how many items their ids, etc...

Reply:
Here should be code like: cm.updateMenuItem(). configureMapMenu shouldn't be stored, it's 1 method